bootstrap/src/lib/cache/index.ts

```diff
@@ -56,54 +61,57 @@ const defaultCacheBuilder = () => {
   }
 }
 // Options without sensitive data
-const redactOptions = (options) => ({
+export const redactOptions = (options: CacheOptions) => ({
   ...options,
   cacheOptions:
     options.cacheOptions.type === 'redis'
       ? redis.redactOptions(options.cacheOptions)
       : local.redactOptions(options.cacheOptions),
 })
 
-const withCache = async (execute, options) => {
+export const withCache = async (
+  execute: ExecuteWrappedResponse,
+  options: CacheOptions = defaultOptions(),
+) => {
   // If no options read the env with sensible defaults
-  if (!options) options = defaultOptions()
+  if (!options) options
```

Comment:
Please revert this line, just so we don't mess up something around cache lifecycle, as it is not so robust. Looks harmless, I know, but still :) We have some caching work incoming, so we'll make any changes there where it's going to be properly tested.

```suggestion
  if (!options) options = defaultOptions()
```